nx/include/switch/applets/psel.h

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file psel.h
+ * @brief Wrapper for using playerSelect (user selection applet).
+ * @author XorTroll
+ * @copyright libnx Authors
+ */
+#pragma once
+#include "../types.h"
+#include "../services/applet.h"
+#include "../services/acc.h"
+
+// playerSelect UI modes.
+typedef enum {
+    PselUiMode_SelectUser                           = 0,  ///< Simple user selection (new users cannot be created).
+    PselUiMode_UserCreation                         = 1,  ///< Only user creation (the user is later returned).
+    PselUiMode_EnsureNsaAvailable                   = 2,  ///< EnsureNsaAvailable
+    PselUiMode_IconEditor                           = 3,  ///< IconEditor
+    PselUiMode_NicknameEditor                       = 4,  ///< NicknameEditor
+    PselUiMode_ForStarter                           = 5,  ///< Mode "starter" uses to register the console's first user on the initial setup
+    PselUiMode_NetworkServiceAccountRegistration    = 8,  ///< NetworkServiceAccountRegistration
+    PselUiMode_NintendoAccountNnidLinker            = 9,  ///< NintendoAccountNnidLinker
+    PselUiMode_LicenseRequirementsForNetworkService = 10, ///< LicenseRequirementsForNetworkService
+    PselUiMode_NaLoginTest                          = 12, ///< NaLoginTest
+} PselUiMode;
+
+// UI settings for playerSelect.
+typedef struct {
```

Comment:
Official sw stores structs within UiSettings, update this struct for that (one of those funcs also writes a titleID to UiSettings, FWIW).

Reply:
What exactly do you mean with "stores structs within UiSettings", that its content is split into different structs?

Reply:
Yes, same as official sw.